### ISSUE TITLE
Include option to output to .xlsx

### ISSUE THREAD

**Issue by buidav** [cisagov/ScubaGear#1355]
# 💡 Summary #

This issue to explore and implement outputting ScubaGear runs to `.xlsx`
Adding `.xlsx` allows for additional features to be added to the spreadsheet outputs.
Such as multiple tabs, graphs, filters etc.  

## Motivation and context ##
- To provide more flexibility of output options

## Implementation notes ##
- Requires an additional PowerShell module to be included as a dependency [ImportExcel](https://www.powershellgallery.com/packages/ImportExcel/7.8.4)
- Add code in `Orchestrator.psm1` to use `Export-Excel` as well.

## Acceptance criteria ##
- ScubaGear outputs to `.xlsx` as an option for spreadsheet output
  - `ScubaResults.xlsx` or some other file is output.

**Comment by gdasher** [comment]
I agree it would be better to make this an option and not require people use excel.  Some workflows may make more sense using csv for integrating into custom tooling or the like.  That being said, I do think adding ability to export to excel natively would be useful for some scenarios.
